R/plotLearnerPredictionPlotly.R

```diff
@@ -0,0 +1,320 @@
+#' @param show.point [\code{logical(1)}]\cr
+#'   Show the input data point?
+#'   Default is \code{TRUE}.
+#' @param show.legend [\code{logical(1)}]\cr
+#'   For classification: Show the legend of right classified points?
+#'   Default is \code{TRUE}.
+#' @param show.colorbar [\code{logical(1)}]\cr
+#'   Show the colorbar?
+#'   Default is \code{TRUE}.
+#' @param pointsize [\code{numeric(1)}]\cr
+#'   Pointsize for ggplot2 \code{\link[ggplot2]{geom_point}} for data points.
+#'   Default is 2.
+#' @param point.col = NULL [\code{character(1)}]\cr
+#'   For classification: Set points colors. The color vector muss have either the same length with
+#'   the number of classes of response variable, or just a single color.
+#'   For regression: Only single color is supported.
+#'   Colors are accepted in several different ways, see "Color Specification" section in \code{\link[graphics]{par}}.
+#'   Defaul is \code{NULL}.
+#' @param point.alpha [\code{numeric(1)}]\cr
+#'   For classification: Set the transparancy of prediction point for classification 3D plots with value from 0 to 1.
+#'   Default is 1.
+#' @param err.mark [\code{character(1)}]:
+#'   For classification: Either mark error of the model on the training data (\dQuote{train}) or
+#'   during cross-validation (\dQuote{cv}) or not at all with \dQuote{none}.
+#'   Default is \dQuote{train}.
+#' @param err.size [\code{numeric(1)}]\cr
+#'   For classification: Set misclassified point size.
+#'   Default is \code{pointsize}.
+#' @param err.col [\code{character(1)}]\cr
+#'   For classification: Set the colors of missclassified data points.
+#'   Default value is \code{NULL} with black color.
+#' @param err.alpha [\code{numeric(1)}]\cr
+#'   For classification: Set the transparancy of missclassified data points.
+#'   Default value is \code{point.alpha}.
+#' @template arg_prettynames
+#' @param show.bounding [\code{logical(1)}]\cr
+#'   For classification: Show the bounding region?
+#'   Default is \code{TRUE}.
+#' @param bounding.alpha [\code{numeric(1)}]\cr
+#'   For \code{show.bounding = TRUE}: Set the transparancy of bounding point.
+#'   Default is 0.5.
+#' @return The plotly object.
+#' @importFrom data.table dcast
+#' @importFrom RColorBrewer brewer.pal
+#' @importFrom plotly plot_ly add_trace %>% layout toRGB hide_colorbar hide_legend
+#' @export
+plotLearnerPredictionPlotly = function(learner, task, features = NULL, measures, cv = 10L,  ...,
+                                       gridsize, show.point = TRUE, show.legend = TRUE, show.colorbar = TRUE,
+                                       pointsize = 2, point.col = NULL, point.alpha = 1,
+                                       err.mark = "train", err.size = pointsize, err.col = NULL, err.alpha = point.alpha,
+                                       pretty.names = TRUE, show.bounding = TRUE, bounding.alpha = 0.5) {
+  learner = checkLearner(learner)
+  assert(
+    checkClass(task, "ClassifTask"),
+    checkClass(task, "RegrTask")
+  )
```

Comment:
Sorry, I have tried to use `assertClass`, i.e.
```
assert(
    assertClass(task, "ClassifTask"),
    assertClass(task, "RegrTask")
)
```
But it pass successfully just when both of them are TRUE, whereas one of them is enough, and not possible both are TRUE.